src/lib/labeller.ts

```diff
@@ -174,7 +174,16 @@ class StepLabeller implements StepMatcher<string> {
   }
 
   cumsum(step: Readonly<S.CumSumStep>) {
-    return `Compute cumulated sum of "${step.valueColumn}"`;
+    // For retrocompatibility with old configurations
+    if (step.valueColumn) {
+      return `Compute cumulated sum of "${step.valueColumn}"`;
+    }
+
+    if (step.toCumSum.length === 1) {
+      return `Compute cumulated sum of "${step.toCumSum[0][0]}"`;
+    } else {
+      return `Compute cumulated sum of ${formatMulticol(step.toCumSum.map(a => a[0]))}`;
+    }
```

Comment:
It would be nice to add test case here: https://github.com/ToucanToco/weaverbird/blob/master/tests/unit/labeller.spec.ts#L668
for this new case